src/main/webapp/WEB-INF/view/register.jsp

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
The formatting of the file needs to be fixed. The opening tag and closing tag should be at the same indent level. There are tools such as https://github.com/victorporof/Sublime-HTMLPrettify that can automatically do this for you.

Reply:
okay fixed!